### DISCUSSION TITLE
Ability to Configure Udp Buffer Size

### DISCUSSION THREAD

**Discussion by noahrose18** [nasa/fprime#2362] [Ideas]
On line 55 of fprime/Drv/Udp/UdpComponentImpl.cpp, there is a buffer that is allocated to a size of 1024 bytes. I am wondering if this could be pulled out and turned into a tunable parameter, so that I can for instance increase the size of this buffer without having to fork off of the fprime repo or autopatch everytime.

**Comment by LeStarch** [comment]
Yes, we should break out those hard-coded allocations into a parameter of the configure function.  That way users can set this should they need to.

**Reply by LeStarch** [reply]
I made this into an issue to track.

**Reply by noahrose18** [reply]
Thanks!!
